02-oop-inheritance/Movie.js

```diff
@@ -0,0 +1,95 @@
+var MovieObserver = (function () {
+  var playing = function (name) {
+        console.log(name + ' is playing...');
+      },
+      stopped = function (name) {
+        console.log(name + ' is stopped...');
+      };
+  
+  return {
+    playing: playing,
+    stopped: stopped
+  };
+})();
+
+var DownloadableMovie = (function () {
```

Comment:
This doesn't really extend Movie. This is a singleton and we need another class constructor extending the Movie constructor. You could extend the prototype from the first one to the latter one and add the download method.